### ISSUE TITLE
Feature for custom delay time

### ISSUE THREAD

**Issue by Kowls0419** [surfbryce/beautiful-lyrics#223]
I'm using airplay to play my Spotify music on my Apple HomePod and because of the 1-2 second delay with my Mac, the lyrics don't sync up. It would be great if I could set up a custom delay time setting and be able to switch between default and custom sync time.

**Comment by surfbryce** [comment]
@Kowls0419 Yeah I've experienced this myself and somebody even made an issue in the past with almost this exact description but I can't find it so I'll be using this. Somebody also had requested the same thing but didn't explain their reasoning so I'm just going to use this post for it and mark theirs as a duplicate.

Can't guarantee that this'll be done anytime soon, it'll probably just be a super simple button that can add/subtract seconds in terms of delay.

**Comment by itswilliboy** [comment]
Any progress on this?
